frontend/src/layout/navigation-3000/sidepanel/sidePanelLogic.tsx

```diff
@@ -58,6 +59,8 @@ export const sidePanelLogic = kea<sidePanelLogicType>([
                     tabs.push(SidePanelTab.Docs)
                 }
 
+                tabs.push(SidePanelTab.Activation)
```

Comment:
Should we be conditionally rendering this only if they have not completed all the steps?

Reply:
See the TODO list. I tried it but kea really exploded as soon as I connected the logic to make this determination...

Reply:
Sorry, should read the descriptions first 🙈